app/controllers/TaskController.scala

```diff
@@ -214,10 +214,21 @@ class TaskController @Inject() (implicit val env: Environment[User, SessionAuthe
                     Logger.error("No timestamp given for a new label")
                     None
                 }
+
+                var calculatedStreetEdgeId = streetEdgeId;
```

Comment:
I try to use `var`s as infrequently as possible in Scala, opting for `val`s instead (bc they are immutable). Once way to do that here would be to replace the `if` statement with
`val calculatedStreetEdgeId: Int = possibleStreetEdgeId.getOrElse(streetEdgeId)`

Reply:
ok! will do.

Reply:
Mikey and I discussed this on slack, it looks like this ```for (point: LabelPointSubmission <- label.points)``` shouldn't exist because there should only be one entry in label.points. We decided to leave this code as is.